src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx

```diff
@@ -176,24 +176,24 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
   const _asset = useAppSelector(state => selectAssetById(state, foundPool?.assetId ?? ''))
```

Comment:
might as well update `_asset` to `_poolAsset` as well for clarity

Reply:
absolutely - 519083356b